templates/server.yaml

```diff
@@ -32,8 +32,11 @@ spec:
             - sh
             - -c 
             - >
+              {{- if .Values.server.envFiles }}
               export $(xargs < {{ join " " .Values.server.envFiles }});
-              /root/configs/docker/cmd.sh
+              {{- end }}
+              env;
```

Comment:
this will expose the secrets

Reply:
Might have forgotten the `env;` from debugging the deployment